CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## 2.13.0-rc.0 / 2019-09-25
+
+* [CHANGE] Metrics: Change prometheus_sd_configs_failed_total to Gauge #5254
+* [CHANGE] Service discovery: add new node address types for kubernetes. #5902
+* [CHANGE] Metrics format: update parsing to disallow new Go 1.13 float formats. #5984
```

Comment:
not sure this needs to be in the changelog since it doesn't change anything.

Reply:
doesn't this mean that the exporters should change how they export floats?

Reply:
No, there's no changes here. This doesn't need to be mentioned.

Reply:
removed